CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing to Serde
+
+Serde welcomes contribution from everyone. Here are the guidelines if you are
+thinking of helping us:
+
+## Contributions
+
+Contributions to Serde or its dependencies should be made in the form of GitHub
+pull requests. Each pull request will be reviewed by a core contributor
+(someone with permission to land patches) and either landed in the main tree or
+given feedback for changes that would be required. All contributions should
+follow this format, even those from core contributors.
+
+Should you wish to work on an issue, please claim it first by commenting on
+the GitHub issue that you want to work on it. This is to prevent duplicated
+efforts from contributors on the same issue.
+
+## Pull Request Checklist
+
+- Branch from the master branch and, if needed, rebase to the current master
+  branch before submitting your pull request. If it doesn't merge cleanly with
+  master you may be asked to rebase your changes.
```

Comment:
We've been talking in #318 about rebasing vs merging. Homu can be instructed to rebase and autosquash. The advantage of merging instead of rebasing is easier reviewing